azure-pipelines-samples.yml

```diff
@@ -6,6 +6,7 @@ resources:
 variables:
   SamplesBin: SamplesBin
   WINDOWS_WINMD: C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.18362.0\Windows.winmd
+  WindowsTargetPlatformVersion: 10.0.18362.0
```

Comment:
Why do we need to override the target platform version in our build pipeline? Shouldn't we be building it in the way that our samples are configured externally?

Reply:
This is to give a way where we can configure the target platform for all samples in one place. In the future, we don't need to update the sample, but change the sdk version here.

---
In reply to: [409810612](https://github.com/microsoft/Windows-Machine-Learning/pull/310#discussion_r409810612) [](ancestors = 409810612)

Reply:
But people that are are using and building the samples on their own will still need the windows target platform version attribute to be set correctly in the project file right? So we'll still need to update all the project files for them.

Reply:
oh I understand. Sounds good!